Off-Top-App/off-top-java
# OT-86 | Create a GET request for logging in

# What was done
- Created a get request that retrieved all of the user's data 
# What you will Need to Test
- Connected toa local Mysql database
- Postman
# How to Test
**Testing in POSTMAN**
- Run the spring boot application. 
- Open another terminal and run `mysql -u root -p` then type in the password
 - Copy and paste this SQL statement ```INSERT INTO `offtop`.`user` (`age`, `city`, `created_at`, `email`, `first_name`, `gender`, `last_name`, `password`, `professional`, `username`) VALUES ('23', 'los angeles', Now(), 'john.doe@email.com', 'John', 'male', 'Doe', 'abc123', 'student', 'jdoe')``` This will create an entry in your MySQL database
- Now go to Postman and create a get request with the URL  `http://localhost:9000/user/john.doe@email.com`
- The returned value should be similar to the JSON shown below
```
{
    "Id": 2,
    "firstName": "John",
    "lastName": "Doe",
    "city": "los angeles",
    "age": 23,
    "gender": "male",
    "professional": "student",
    "email": "john.doe@email.com",
    "username": "jdoe",
    "password": "abc123",
    "createdAt": "2020-04-03 03:55:28",
    "deletedAt": null,
    "id": 2
}
```
